demos/client-example-server/src/app.ts

```diff
@@ -31,23 +31,10 @@ export function getEnvFile(filename = '.env', filepath = './') {
     return path.join(filepath, `${filename}.${env}`)
 }
 
-enum ProsopoVerificationType {
```

Comment:
Why has this been removed? It's needed to allow the tests to pass in captcha-private